docs/swagger/sota-device_registry.yml

```diff
@@ -238,7 +238,39 @@ paths:
             type: array
             items:
               $ref: '#/definitions/uuid'
-
+  /device_groups/{groupId}/devices/{deviceId}:
+    post:
+      description: add a device to a group
+      parameters:
+      - name: groupId
+        description: The uuid of an existing group
+        in: path
+        required: true
+        type: string
```

Comment:
Maybe create a new swagger definition and state that a groupId should be a uuid-thingie.